registration/templates/toc.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'base.html' %}
+
+{% load i18n %}
+
+{% block content %}
+<h1> Terms of Condition </h2>
```

Comment:
```suggestion
<h1> Terms and Conditions </h2>
```
or [Terms of service](https://en.wikipedia.org/wiki/Terms_of_service)

Reply:
I chose `Privacy Policy` now.